tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/HotChocolateCommon.cs

```diff
@@ -142,17 +146,93 @@ private static string ConstructErrorMessage(List<IError> executionErrors)
+                        {
+                            if (location.TryDuckCast<ErrorLocationStruct>(out var locationProxy))
+                            {
+                                joinedLocations.Add($"{locationProxy.Line}:{locationProxy.Column}");
+                            }
+                        }
+
+                        eventAttributes.Add(new KeyValuePair<string, object>("locations", joinedLocations.ToArray()));
                     }
 
-                    ConstructErrorLocationsMessage(builder, tab, executionError.Locations);
                     builder.AppendLine($"{tab}}},");
+
+                    var path = executionError.Path;
+                    if (path.Name != null)
+                    {
+                        var pathAttribute = new List<string>();
+                        pathAttribute.Add(path.Name);
+                        eventAttributes.Add(new KeyValuePair<string, object>("path", pathAttribute.ToArray()));
+                    }
+
+                    var code = executionError.Code;
+                    if (code != null)
+                    {
+                        eventAttributes.Add(new KeyValuePair<string, object>("code", code));
+                    }
+
+                    var exception = executionError.Exception;
+                    if (exception != null)
+                    {
+                        eventAttributes.Add(new KeyValuePair<string, object>("stacktrace", exception.StackTrace));
+                    }
+
+                    var extensions = executionError.Extensions;
+                    if (extensions != null)
+                    {
+                        var configuredExtensions = Tracer.Instance.Settings.GraphQLErrorExtensions;
+
+                        var keys = extensions.Keys.ToList();
+                        for (int j = 0; j < keys.Count; j++)
+                        {
+                            if (configuredExtensions.Contains(keys[j]))
+                            {
+                                var key = keys[j];
+                                var value = extensions[key];
+
+                                if (value == null)
+                                {
+                                    value = "null";
+                                }
+                                else if (value is Array array)
+                                {
+                                    var stringArray = new string[array.Length];
+                                    for (int k = 0; k < array.Length; k++)
+                                    {
+                                        stringArray[k] = array.GetValue(k)?.ToString() ?? "null";
+                                    }
+
+                                    value = stringArray;
+                                }
+                                else if (!(value is int || value is double || value is float || value is bool))
```

Comment:
If you're testing for all primitives then the complete set should be the following (see see [this StackOverflow link](https://stackoverflow.com/a/1130705) as a reference):
- `sbyte`
- `byte`
- `short`
- `ushort`
- `int`
- `uint`
- `long`
- `ulong`
- `float`
- `double`
- `decimal`
- `bool` (since you're trying to keep bool as-is)